data/json/furniture_and_terrain/furniture-seats.json

```
@@ -204,5 +204,218 @@
       "sound_fail": "whump.",
       "items": [ { "item": "2x4", "count": [ 1, 3 ] }, { "item": "nail", "charges": [ 2, 6 ] }, { "item": "splinter", "count": 1 } ]
     }
+  },
+  {
+    "type": "furniture",
+    "id": "f_floralprint_armchair",
+    "name": "floral print armchair",
+    "symbol": "H",
+    "description": "A more comfortable way of sitting down.  This reminds you of a nursing home.",
```

```suggestion
    "description": "A more comfortable way of sitting down.  It reminds you of a nursing home.",
```

Sounds more natural to me this way, feel free to leave it as is if you disagree.